source/nodejs/adaptivecards-designer/package.json

```diff
@@ -47,6 +47,7 @@
 		"@typescript-eslint/eslint-plugin": "^3.4.0",
 		"@typescript-eslint/parser": "^3.4.0",
 		"cpy-cli": "^3.1.1",
+		"dotenv-webpack": "^1.7.0",
```

Comment:
>"dotenv-webpack": "^1.7.0", [](start = 2, length = 27)

Curious: Is this actually supposed to be devDependencies or dependencies?

Reply:
When a user consumes the `adaptivecards-designer` via npm, the `.env` file won't be active for them, their root project takes care of it, if it requires for them. So we don't need to pack it in production npm package.

In our case the `adaptivecard-sites` has tight integration with `adaptivecard-designer`, so at release time `.env` helps to simplify the deployments for https://adaptivecards.io site. Internal projects most of them keeps dependencies in dev, so I guess it's fine for internal use also.